external-crates/move/Cargo.toml

```diff
@@ -191,10 +192,10 @@ move-vm-runtime-v0 = { path = "move-execution/v0/crates/move-vm-runtime" }
 move-vm-runtime-v1 = { path = "move-execution/v1/crates/move-vm-runtime" }
 move-vm-runtime-v2 = { path = "move-execution/v2/crates/move-vm-runtime" }
 # move-vm-runtime-$CUT = { path = "move-execution/$CUT/crates/move-vm-runtime" }
-move-stdlib-v0 = { path = "move-execution/v0/crates/move-stdlib" }
-move-stdlib-v1 = { path = "move-execution/v1/crates/move-stdlib" }
-move-stdlib-v2 = { path = "move-execution/v2/crates/move-stdlib" }
-# move-stdlib-$CUT = { path = "move-execution/$CUT/crates/move-stdlib" }
+move-stdlib-natives-v0 = { path = "move-execution/v0/crates/move-stdlib-natives" }
+move-stdlib-natives-v1 = { path = "move-execution/v1/crates/move-stdlib-natives" }
+move-stdlib-natives-v2 = { path = "move-execution/v2/crates/move-stdlib-natives" }
+# move-stdlib-natives-$CUT = { path = "move-execution/$CUT/crates/move-stdlib-natives" }
```

Comment:
Ditto the comment on the previous PR about not adding workspace dependencies for the versioned crates.